editor/src/core/shared/string-utils.ts

```diff
@@ -3,7 +3,8 @@ export function capitalize(value: string): string {
 }
 
 export function firstLetterIsLowerCase(value: string): boolean {
-  return value.length > 0 && isLowerCase(value[0])
+  const charCode = value.charCodeAt(0)
+  return !isNaN(charCode) && charCode >= 97 && charCode <= 122
```

Comment:
just for clarity would it be possible to use `'a'` and `'z'` instead of their numeric charcodes here? maybe something like `'a'.charCodeAt(0)` and put it into a const 🤔

Reply:
Yeah, that's a good idea.

Reply:
Fixed in 045fe83f1.